client/src/pages/Home.tsx

```diff
@@ -0,0 +1,128 @@
+import { Collection } from '../lib/types';
+import { Link } from 'react-router-dom';
+
+const collections: Collection[] = [
+  {
+    name: 'Placeholder',
+    imageSrc: '/images/1200x1200.svg',
+    imageAlt: 'Placeholder',
+    description: 'Placeholder description',
+  },
+  {
+    name: 'Placeholder',
+    imageSrc: '/images/1200x1200.svg',
+    imageAlt: 'Placeholder',
+    description: 'Placeholder description',
+  },
+  {
+    name: 'Placeholder',
+    imageSrc: '/images/1200x1200.svg',
+    imageAlt: 'Placeholder',
+    description: 'Placeholder description',
+  },
+];
+
+export default function Home(): JSX.Element {
+  return (
+    <div className="bg-white">
+      {/* Hero section */}
+      <div className="relative bg-gray-900">
+        {/* Decorative image and overlay */}
+        <div aria-hidden="true" className="absolute inset-0 overflow-hidden">
+          <img
+            src="/images/2716x1600.svg"
+            alt="Placeholder"
+            className="h-full w-full object-cover object-center"
+          />
+        </div>
+        <div
+          aria-hidden="true"
+          className="absolute inset-0 bg-gray-900 opacity-50"
+        />
+
+        <div className="relative mx-auto flex max-w-3xl flex-col items-center px-6 py-32 text-center sm:py-64 lg:px-0">
+          <h1 className="text-4xl font-bold tracking-tight text-white lg:text-6xl">
+            Text
+          </h1>
+          <p className="mt-4 text-xl text-white">
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Harum modi
+            molestias necessitatibus iste ex a, est voluptatum doloremque
+            nostrum consequuntur voluptate porro asperiores, dolor libero, eius
+            fugit ratione repellendus similique.
+          </p>
+        </div>
+      </div>
+
+      <main>
+        {/* Collection section */}
```

Comment:
I hope that as things evolve you will extract all this into separate components. This file is getting huge!